synfig-studio/src/gui/states/state_bone.cpp

```diff
@@ -245,6 +263,8 @@ void
 StateBone_Context::reset()
 {
 	active_bone = -1;
+	set_id(_("NewSkeleton"));
+	set_bone_width(Distance(6.6,App::distance_system)); // default width
```

Comment:
6.6 :)

Reply:
This issue is still present in current code.

Reply:
I fixed this tooo